lib/DojotHandler.js

```diff
@@ -9,8 +9,10 @@ const util = require('util');
 
 const ZMQ_PORT = 5555;
 
+const TAG = {filename: "flownode/dojotHandler"};
+
 function checkField(target, field, message) {
-  if (!target.hasOwnProperty(field)) {
+  if (!target[field]) {
```

Comment:
Just a side note: there is a warning in eslint indicating that this call is closer to the original. It should be something like `Object.prototype.hasOwnProperty.call(target, field)`. Check https://github.com/eslint/eslint/issues/7071#issuecomment-245377924 for further info.

Reply:
It's nice to know